interfaces/sourcegen/src/sourcegen/clib/template_header.h.in

```diff
@@ -19,14 +19,15 @@
  *  %Cantera build process. Do not manually edit.
  *
  *  @warning  This module is an experimental part of the %Cantera API and
- *      may be changed or removed without notice.
+ *      may be changed without notice.
  */
 
 // This file is part of Cantera. See License.txt in the top-level directory or
 // at https://cantera.org/license.txt for license and copyright information.
 
 #ifndef {{ guard }}
 #define {{ guard }}
+#pragma once
```

Comment:
I don't think we should have both a standard header guard and `#pragma once`. Whether we want to replace the more complex header guards with `#pragma once` is a separate question (and definitely not one for this PR).